### DISCUSSION TITLE
Do we define [ObsoleteAttribute]-ing as a breaking change?

### DISCUSSION THREAD

**Discussion by antonfirsov** [SixLabors/ImageSharp#1454] [Q&A] [answered]
I think this is matter of preference, and not trivial.

Some argue very strongly that obsoletion is a breaking change, since when user builds with  `/warnaserror` it may result in unexpected build errors on a package update, and needs developer intervention.

In any case, such a move should be fine with a minor release bump IMO, we just need to decide if we apply the `breaking` label on the PR or not.

**Comment by JimBobSquarePants** [comment, marked as answer]
Yep. I think adding obsolete should certainly  be marked with `breaking`. We’ll use it sparingly and try to proxy updates where possible

**Reply by antonfirsov** [reply]
Cool, great that we agree!
